projects/js-packages/charts/src/components/pie-semi-circle-chart/pie-semi-circle-chart.tsx

```diff
@@ -178,25 +180,25 @@ const PieSemiCircleChartInternal: FC< PieSemiCircleChartProps > = ( {
 		);
 	}
 
+	// Calculate chart dimensions
+	//TODO: we might want to accept height as a prop in the future, because the height of container might not always be enough.
 	const height = width / 2;
-	const pad = 0.03;
-
-	// Use padding for the overall chart dimensions
-	const chartWidth = width - pad * 2;
```

Comment:
Taking away pad*2 makes no sense here 😓

Reply:
Yep, good call. It looks like this was carried over from the regular pie chart where this was likely accounting for padding on both left + right sides. However, for a semi-circle chart horizontal centering really is what matter. Loving the tidy up, much cleaner.

Reply:
Yeah I think we just mixed the padding between segments with the padding for the chart itself here.